realhf/experiments/common/ppo_exp.py

```diff
@@ -188,6 +188,18 @@ class PPOConfig(CommonExperimentConfig):
     :type dataset: PromptOnlyDatasetConfig
     :param ppo: Configuration for the PPO algorithm.
     :type ppo: PPOHyperparameters
+    :param actor_train_n_mbs: Number of minibatches for TrainActor.
+    :type actor_train_n_mbs: int
+    :param critic_train_n_mbs: Number of minibatches for TrainCritic.
+    :type critic_train_n_mbs: int
+    :param actor_gen_n_mbs: Number of minibatches for Rollout.
+    :type actor_gen_n_mbs: int
+    :param critic_inf_n_mbs: Number of minibatches for InfValues.
+    :type critic_inf_n_mbs: int
+    :param rew_inf_n_mbs: Number of minibatches for InfReward.
+    :type rew_inf_n_mbs: int
+    :param ref_inf_n_mbs: Number of minibatches for InfRef.
+    :type ref_inf_n_mbs: int
```

Comment:
Consider merge these and `AllocationConfig` into an `RPCConfig`/`MFCConfig` to avoid write a config param for each RPC in experiment config file? Maybe in the future we will need other parameters that are different across different RPCs then we will need another 6 parameters here.

Reply:
BTW we need to decide whether to use MFC or RPC in our experiment config codes, now it is a bit confusing.

Reply:
> BTW we need to decide whether to use MFC or RPC in our experiment config codes, now it is a bit confusing.

Since changing all name `RPC` to `MFC` will change a lot of variable names, I added notes in definition of `MFCDef` and property function `rpcs` in experiment configs to clarify this problem.

Reply:
> Consider merge these and `AllocationConfig` into an `RPCConfig`/`MFCConfig` to avoid write a config param for each RPC in experiment config file? Maybe in the future we will need other parameters that are different across different RPCs then we will need another 6 parameters here.

Change `AllocationConfig` into `MFCConfig`.